dali/operators/reader/loader/nemo_asr_loader_test.cc

```diff
@@ -106,8 +106,9 @@ TEST(NemoAsrLoaderTest, WrongManifestPath) {
 void tempfile(std::string& filename, std::string content = "") {
   int fd = mkstemp(&filename[0]);
   ASSERT_NE(-1, fd);
-  if (!content.empty())
-    write(fd, content.c_str(), content.size());
+  if (!content.empty()) {
+    ASSERT_EQ(write(fd, content.c_str(), content.size()), content.size());
```

Comment:
Won't it raise a warning for comparing signed with unsigned (content.size())?

Reply:
I don't see any.

Reply:
What I see it uses the function that might cast booth sizes to just signed int.